spec/forms/other_income_form_spec.rb

```diff
@@ -1,11 +1,13 @@
 require "rails_helper"
 
-RSpec.describe "other_income", type: :feature do
+RSpec.describe "other_income", :stub_cfe_calls, type: :feature do
   let(:assessment_code) { :assessment_code }
   let(:feature_flags) { {} }
 
   before do
-    set_session(assessment_code, "level_of_help" => "controlled", "feature_flags" => feature_flags)
+    set_session(assessment_code, "level_of_help" => "controlled", "feature_flags" => feature_flags,
```

Comment:
remove this and make it into a feature spec that the runs through the flow until

Reply:
we actually have so many of these form specs so instead of getting into refactoring this one now, maybe we want to take a look at the together as a bigger test refactor. I have removed the changes to the session and the tests still pass with flag on and off so would like to leave them as is for now.